tools/dockerfile-add-scanner/cli/licenses.go

```diff
@@ -0,0 +1,58 @@
+package cli
```

Comment:
Should this file have some Copyright (the original one from where you coped)?
A License SPDX?

Reply:
Yeah, good point. They all should. Just added them.